### PR TITLE
feat: add presentations page

### PR DESCRIPTION
**Pull request by beeme1mr**
## This PR

- adds a page that lists presentations

### Notes

This is just a list of presentations I could find quickly. Please let me know what I've missed and I would be happy to add them.

We should also create a shareable PowerPoint template that can be shared.

### CODE REVIEW

**Review by toddbaert — APPROVED** [review]
I'm good with this, but let's wait for others for just a bit to see if they have any recordings, etc.

I'll ping a couple people who might (not sure): @agardnerIT @lukas-reining @thomaspoignant @moredip

**Comment by toddbaert** [comment]
There's also 2 episodes of @dabeeeenster 's podcast: https://www.flagsmith.com/podcast/openfeature and https://www.flagsmith.com/podcast/openfeature-pt-2

**Comment by toddbaert** [comment]
There's also an episode of [PurePerformance](https://www.spreaker.com/episode/introducing-openfeature-stepping-into-the-footsteps-of-opentelemetry-with-mike-beemer-and-todd-baert--50043067) you and I did.

File: presentations.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Presentations
+
+This page showcases presentation given by the OpenFeature community, providing links to past presentations, and offering resources for community members who wish to contribute to future presentations.
```
Comment (toddbaert) [inline presentations.md:3]:
```suggestion
This page showcases presentations and interviews given by the OpenFeature community, providing links, and offering resources for community members who wish to contribute to future presentations.
```

File: presentations.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Previous presentations
+
+This section provides an overview of all the presentations given by the OpenFeature community, including links to recordings and slides where available.
```
Comment (toddbaert) [inline presentations.md:7]:
```suggestion
This section provides an overview of presentations and interviews given by OpenFeature community members, including links to recordings and slides where available.
```

**Review by DavidPHirsch — APPROVED** [review]

**Review by AloisReitbauer — APPROVED** [review]

**Pull request merged**